idea/src/org/jetbrains/kotlin/idea/kdoc/KDocRenderer.kt

```diff
@@ -199,6 +198,18 @@ object KDocRenderer {
         }
     }
 
+    private fun markdownToHyperLink(markdown: String): String? =
```

Comment:
Do we to need to check the markdown structure here? Maybe just `parseMarkdown(markdown).toHtml()`?

Reply:
I think that `@see` block must support only links, other markdown looks like error